pennylane/return_types.py

```diff
@@ -19,17 +19,153 @@
 
 
 def enable_return():
-    """Function that turns on the new return type system."""
+    """Function that turns on the new return type system. The new system guarantees intuitive return types such that a
+    sequence (e.g., list or tuple) is returned based on the `return` statement of the quantum function. This system
+    avoids the creation of ragged arrays, where multiple measurements are stacked together.
+
+    **Example**
+
+    The following example shows that for multiple measurements the current PennyLane system is creating a ragged tensor.
+
+    ```python
```

Comment:
Ohhh @rmoyard @antalszava it's because this is ReST, so we need to do `.. code-block:: python` for python blocks, and just `>>>` for pycon blocks 😆